test/unit/test-config.cpp

```diff
@@ -133,8 +133,6 @@ TEST_CASE("Config Driven Solver", "[config][Serial]")
   auto jsonstream = PreprocessFile(filename.c_str());  // Apply custom palace json
   auto config = json::parse(jsonstream);
 
-  using namespace Catch::Matchers;
```

Comment:
Why did you remove this? Aren't the tests cleaner with `using namespace`?

Reply:
You are right, this can make the code more cluttered. I personally like to make dependancies more visible, but I am open to declaring the namespace if the team prefers it. I also, wanted to make the function calls consistent across all test files, including `test-vector.cpp` in which the namespace hadn't been declared. That said, I am totally fine with either approaches.

Reply:
I agree with making the dependencies explicit. In this particular case, I chose to hide the namespace because this function is used all the time, so developers would have to become familiar with it quikcly, but I am find with keeping it more verbose.